docs/_documentations/importing-existing-projects.md

```diff
@@ -21,17 +21,19 @@ Add a project from an existing Git repository:
 
 ## Requirements for importing an existing project not created in Codewind or through Appsody or Odo
 
-Local projects are built and run on Docker. To import your project to a local Codewind instance, you must include a Dockerfile. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/).
-**Note:** The Dockerfile must contain an `EXPOSE` instruction to point to the port that the project runs on.
+Local projects are built and run on Docker. To import your project to a local Codewind instance, you must include a Dockerfile. For more information about Dockerfiles, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/). **Note:** The Dockerfile must contain an `EXPOSE` instruction to point to the port that the project runs on.
 
 Remote projects are built and run on Kubernetes. To import your project to a remote Codewind instance, you must include a Dockerfile and a Helm chart. For more information about Dockerfiles and Helm charts, see [Dockerfile documentation](https://docs.docker.com/engine/reference/builder/) and [Helm chart documentation](https://helm.sh/docs/topics/charts/).
 
+To import your own project into Codewind, follow these same requirements. Make sure to select the **Other (Basic Container)** option as the project type.
+
 ## What kind of projects can I add?
 
-Codewind develops cloud-native microservices. Therefore, each project must be self-sufficient, not reliant on other projects to build. 
+Codewind develops cloud-native microservices. Therefore, each project must be self-sufficient, not reliant on other projects to build. Learn more about projects you can undertake by using:
```

Comment:
I think it sounds weird to say "using... Codewind", "using... Appsody", "using... Odo" here.

Are we trying help users decide what style project projects they want to develop here?

Maybe we should just be explicit such as:  "If you want to do [A], then you should pick Codewind style" etc?

Reply:
@makandre, Thanks for the input, Andrew. Yes, we want to help users determine what projects they want to develop. Basically, we want to drive users to the right information, so letting them know to read up on Codewind, Appsody, odo, whatever they're working on or plan to work on. 

It must be brief, but clear, since users likely shall scan this page. Does that make sense?

I made a new commit so please let me know if you think it's improved and what else must be changed. Thanks!